src/components/Services.astro

```diff
@@ -0,0 +1,63 @@
+---
+import Card from './Card.astro';
+
+const cards = [
```

Comment:
you might add a Interface / Type in here :D 

like 
```ts
interface Card { 
  img: string; 
  description: string 
  }
  
  const cards: Card[] = [...]
  ```

Reply:
That is interesting, because I had that and deleted then :D

Reply:
Okey, I deleted it because it is said that is already defined:

"✖ eslint --fix:

/Users/mateuszkarcz88gmail.com/Documents/Praktyki - Artur/Astro_project/src/components/Services.astro
  4:11  error  'Card' is already defined  no-redeclare

✖ 1 problem (1 error, 0 warnings)

husky - pre-commit hook exited with code 1 (error)
"

Reply:
I added it and the website went down.

Reply:
Name overlaping, now card and inputs have the same format. 
try 
```ts
interface CardInterface { 
...
```

Reply:
I deleted and now I cannot make a "commit & push" because: 
"/Users/mateuszkarcz88gmail.com/Documents/Praktyki - Artur/Astro_project/src/components/Services.astro
  4:11  error  'Card' is already defined  no-redeclare"

I do not get it :/